t/data/openqa/factory/repo/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
why delete this? so people will notice if the test process doesn't clean up after itself?